Yosemite/YosemiteTests/Mocks/CardPresentPayments/MockCardReaderService.swift

```diff
@@ -9,7 +9,7 @@ final class MockCardReaderService: CardReaderService {
     }
 
     var connectedReaders: AnyPublisher<[Hardware.CardReader], Never> {
-        CurrentValueSubject<[Hardware.CardReader], Never>([]).eraseToAnyPublisher()
+        connectedReadersSubject.eraseToAnyPublisher()
```

Comment:
Same comment as above about one connection max

Reply:
Same comment as above as well :) This was a deliberate decision we made a week or two ago. I'm happy to change it though